src/components/Map/Map.tsx

```diff
@@ -82,13 +83,34 @@ export default function Map({ countries, disputedAreas, fcsData, alertData }: Ma
           setCountryData,
           setCountryIso3Data,
           setRegionNutritionData,
-          setIpcRegionData
+          setIpcRegionData,
+          setData
         );
-        setSelectedCountryName(selectedCountryData.properties.adm0_name);
-        mapRef.current?.fitBounds(L.geoJSON(selectedCountryData as GeoJSON).getBounds(), { animate: true });
+        if (data) {
+          setSelectedCountryName(selectedCountryData.properties.adm0_name);
+          mapRef.current?.fitBounds(L.geoJSON(selectedCountryData as GeoJSON).getBounds(), { animate: true });
+        } else {
+          mapRef.current?.fitBounds(
+            [
+              [-90, -180],
+              [90, 180],
+            ],
+            { animate: true }
+          );
+          setSelectedCountryId(null);
```

Comment:
instead of this, I think it's best to just to the same thing the back to global button does (zoomOut 4 I think)

Reply:
i tried to do that, but i got error like useMape() can't be use here, it should be used be descendant of mapRef.

Reply:
`mapRef.current?.zoomOut` should work